docs/contributor/02-01-installation.md

```diff
@@ -9,23 +117,23 @@ In such a setup, you don't need to install Keda Manager. It is installed and man
 
 When you enable the Keda module using your Kyma runtime Kyma custom resource (CR), the Lifecycle Manager downloads the bundled package of the Keda Manager and installs it. Additionally, it applies a sample Keda CR, which triggers Keda Manager to install the Keda module.
```

Comment:
Lets move this subsection ("Lifecycle management of Keda Manager in Kyma") to a separate file in the contributor folder. Lets keep the description, diagram, but please drop the command to enable module via kyma CLI command